src/libs/actions/SignInRedirect.ts

```diff
@@ -12,8 +10,8 @@ import Navigation from '../Navigation/Navigation';
 import * as ErrorUtils from '../ErrorUtils';
 import * as SessionUtils from '../SessionUtils';
 
-let currentIsOffline;
-let currentShouldForceOffline;
+let currentIsOffline: boolean | undefined;
+let currentShouldForceOffline: boolean | undefined;
```

Comment:
Why can't we force these to be non-null? With use of defaults, ect.

Reply:
Values from Onyx which we are assigning later have type `boolean | undefined` so actually there could be a situation where we are assigning undefined to those variables so I would leave it as it is. WDYT ?